src/manifest.json

```
@@ -1,6 +1,6 @@
 {
   "manifest_version": 1,
-  "version": "1.0.4",
+  "version": "1.0.5",
```

```suggestion
  "version": "1.0.6",
```
現在、master ではすでに1.0.5となっているので。。。
すみません🙏

---

最新のpackage.jsonもPushしていただけますでしょうか。
よろしくお願いします。
ほかに変更を加えたファイルがあれば（LICENSE.txtやwebpack.config.tsなど）、一緒にPushしていただけると嬉しいです。